pkg/dogstatsd/server.go

```diff
@@ -119,9 +119,12 @@ func initLatencyTelemetry() {
 type Server struct {
 	// listeners are the instantiated socket listener (UDS or UDP or both)
 	listeners []listeners.StatsdListener
-	// aggregator is a pointer to the aggregator that the dogstatsd daemon
-	// will send the metrics samples, events and service checks to.
-	aggregator *aggregator.BufferedAggregator
+
+	// demultiplexer will receive the metrics processed by the DogStatsD server,
+	// will take care of processing them in multiple routines if possible, and will
```

Comment:
```suggestion
	// will take care of processing them in multiple goroutines if possible, and will
```
or maybe just "in parallel" or "using multiple cores" or something?